config/projects/TEMPLATE.yaml

```diff
@@ -93,7 +93,6 @@ stageable_discovery:
   #     00/
   #       1.tif
   #       2.tif
-  #     01/
```

Comment:
what is this and why is it changing?   Ben in #199 suggests a few different ways we are likely to receive data, and PR #233  and images_jp2_tif  test_data directories have examples of such data.

Not something necessarily to change on this PR but perhaps we need to talk about this?

Reply:
https://github.com/sul-dlss/pre-assembly/issues/243

Reply:
This is a template, I don't see how this question affects anything.  Agree that it doesn't necessarily make sense that it would have changed here, but also not feeling any "loss" over it.